Jenkinsfile

```
@@ -32,6 +36,20 @@ node('vagrant') {
             ])
         }
 
+        stage('Check for tag') {
+            sh "git pull --tags"
+            def tags = sh(returnStdout: true, script: 'git tag -l').trim()
```

```suggestion
            def tags = sh(returnStdout: true, script: 'git tag -l').trim().readLines()
```

I think it would be much better if we use a list of tags here instead of searching for the version in a big string.